### DISCUSSION TITLE
Was unable to log into vercel via github for about 30 minutes, then suddenly started working

### DISCUSSION THREAD

**Discussion by city41** [vercel/community#3271] [Help] [answered]
### Summary

I was trying to log into vercel via github, but it said "bad credentials" every time.

![Screenshot at 2023-07-17 09-37-39](https://github.com/vercel/community/assets/141159/cb98c586-e033-49d0-a177-417ab4956f65)

I tried removing all my vercel and github cookies, as well as incognito mode in Chrome, to the same result. Github itself was working fine, as did other websites where I used github to login.

Just for the heck of it, I tried it one more time. This time github stated vercel wanted permissions, as if I'd never linked vercel and github before. I said ok, and this time I was able to log in.

I disabled my pi-hole, as I noticed some analytic requests were failing in the dev tools. Possibly that allowed me to log in? I am confident I disabled it adequately as the analytic POST requests stopped failing.

Not sure what the issue was, but figured I'd post here in case others are hitting this too.

### Steps to Reproduce

No reliable repro steps unfortunately.

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. You can follow the status page for more updates https://www.vercel-status.com/incidents/590pcktngq7z

**Reply by city41** [reply]
Ah ha, I didn't even think to look at the status page. My bad. I'll look to that in the future. Thanks!
